app/assets/stylesheets/classified_listings.scss

```diff
@@ -255,7 +255,7 @@
       a {
         text-decoration: underline;
         display: inline-block;
-        color: var(--link-branded-color);
+        color: var(--link-brand-color);
```

Comment:
I've changed the name of one variable, that's why there's bunch of random files affected.